components/mailersend/sources/common/base.mjs

```diff
@@ -0,0 +1,28 @@
+import app from "../../mailersend.app.mjs";
+
+export default {
+  props: {
+    app,
+    db: "$.service.db",
+    domainId: {
+      propDefinition: [
+        app,
+        "domainId",
+      ],
+    },
+  },
+  methods: {
+    generateMeta(event) {
+      const { data } = event;
+      return {
+        id: data.id,
+        summary: `${data.type}: ${data.email.subject}`.replace(/(^\w)/g, (m) => m.toUpperCase()),
+        ts: +Date.parse(data.created_at),
```

Comment:
I could not get what `+` sign does here. Is it for purpose? If not can we remove them?

Reply:
This plus serves the purpose of converting the date to timestamp, I haven't tested without it